src/main/java/com/mindsnacks/zinc/classes/data/ZincCatalogs.java

```diff
@@ -81,26 +72,9 @@ public ZincCatalogs(final File root,
      */
     @Override
     public synchronized boolean clearCachedCatalogs() {
-        if (mUpdateScheduled) {
-            throw new ZincRuntimeException("Updates were already scheduled");
-        }
-
         return mFileHelper.emptyDirectory(getCatalogsFolder());
     }
 
-    @Override
-    public void scheduleUpdate() {
-        if (!mUpdateScheduled) {
-            mUpdateScheduled = true;
-
-            mUpdateTimer.schedule(new TimerTask() {
-                @Override public void run() {
-                    updateCatalogsForTrackedSourceURLs();
-                }
-            }, INITIAL_UPDATE_DELAY, UPDATE_FREQUENCY);
-        }
-    }
```

Comment:
I probably wouldn't have removed the whole functionality, *but* don't delay this fix only but that. **At all**.